runelite-client/src/main/java/net/runelite/client/plugins/timetracking/farming/GeomancyMaps.java

```diff
@@ -0,0 +1,307 @@
+/*
```

Comment:
delete this class

Reply:
addressed in 2ffdbad23653acde21afe94f03ff41af4c7af20a